packages/vsce/src/commands/inspectResourceCommandUtils.ts

```diff
@@ -163,8 +160,11 @@ async function loadResources(
   const resources = await resourceContainer.fetchNextPage();
 
   if (resources.length === 0) {
-    const hrn = resourceTypes.map((type) => type.humanReadableNameSingular).join(" or ");
-    const message = CICSMessages.CICSResourceNotFound.message.replace("%resource-type%", hrn).replace("%resource-name%", resourceName).replace("%region-name%", resourceContext.regionName);
+    const hrn = resourceTypes.map((type) => l10n.t(type.humanReadableNameSingular)).join(l10n.t("Or"));
```

Comment:
How does `l10n.t(type.humanReadableNameSingular))` work?? I don't see anything related to this appearing in the bundle file... I think we're better off externalising these values in the meta files (if we are) to avoid this kind of situation.